packages/frontend/src/view/components/AssetWithLogo.tsx

```diff
@@ -6,7 +6,7 @@ import { InlineEllipsis } from './InlineEllipsis'
 
 interface AssetWithLogoProps {
   assetInfo: AssetInfo
-  type?: 'full' | 'regular' | 'small'
+  type?: 'full' | 'regular' | 'small' | 'regularSymbol' //TODO: Figure out a better name for regularSymbol
```

Comment:
I think the idea was regular was what you created `regularSymbol` for :)

Reply:
I thought so too, but regular displays both name and symbol for unknown hashes and we do not want that in `UserQuickActionsTable`

Reply:
Makes sense. Let's shorten it to `symbol` then.